client/src/components/AnimeList/AddToList.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
```

Comment:
You don't need to import react anymore/[havent for a while](https://dev.to/titungdup/you-no-longer-need-to-import-react-from-react-3pbj).